parm/config/config.resources.emc.dyn

```diff
@@ -317,25 +333,33 @@ elif [ $step = "eupd" ]; then
 
     export wtime_eupd="00:30:00"
     if [ $CASE = "C768" ]; then
-      export npe_eupd=315
-      export nth_eupd=14
+      export npe_eupd=480
+      export nth_eupd=6
       if [[ "$machine" = "WCOSS_DELL_P3" ]]; then
 	export npe_eupd=960
         export nth_eupd=7
       fi
+      if [[ "$machine" = "HERA" ]]; then
+        export npe_eupd=150
+        export nth_eupd=40
```

Comment:
what? 40 threads?

Reply:
This is what is already in feature/ops-orion for this job on Hera. There was a need to do this and it would need to be retested to confirm why and if still needed.
https://github.com/NOAA-EMC/global-workflow/blob/feature/ops-orion/parm/config/config.resources.emc.dyn#L321:L324
Didn't need to do this on Orion though.

Reply:
Ok. Thanks for providing history.